ApiTests/Utils/ApiUtils.cs

```diff
@@ -8,16 +8,56 @@ internal class ApiUtils
 
         public static RestResponse SendGetRequest(string resource)
         {
-            //implement a simple get request
-            return null;
+            try
```

Comment:
Would it be worth moving client.Execute(...) to another private method and parametrize the call, so that we won't have the same try-catch in all of the methods here?